examples/gno.land/r/nt/commondao/z_5_f_filetest.gno

```diff
@@ -35,4 +35,4 @@ func main() {
 }
 
 // Error:
-// voting is not enabled
+// should not happen
```

Comment:
These errors should be kept, they are the expected ones.
Same for other "should not happen" cases.